setup.py

```diff
@@ -13,7 +13,7 @@
     author='Vasu Kulkarni',
     author_email='vasu@redhat.com',
     install_requires=[
-        'apache-libcloud',
+        'apache-libcloud>=3.3.0',
```

Comment:
The commit message makes no mention about what bug this version solves, or why we would want to set this *specific* number. I guess that it happens to be the latest version on PyPI, but that might be a coincidence... in general if we're hard-coding version numbers, let's give some clues to the future maintainers about why we're doing it and how safe it might be to bump the numbers in the future, etc.